BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -86,7 +90,10 @@ public actor DefaultTabDataStore: TabDataStore {
 
         do {
             let fileURLs = fileManager.contentsOfDirectory(at: directoryURL)
-            let windowsData = try await parseWindowDataFiles(fromURLs: fileURLs)
+            let windowsData = parseWindowDataFiles(fromURLs: fileURLs)
+            if windowsData.isEmpty {
```

Comment:
It might be just me but I find positive conditions easier to read than negative ones. I'll usually choose between guard and if depending on which will result in the least nested code but in this case they are both one line so in this case opted for an if because it kept the condition positive.
I don't feel strongly about it tho so can change if you don't agree with any of that.